app/models/evaluation.rb

```diff
@@ -48,6 +48,20 @@ class Evaluation < ApplicationRecord
   after_update :update_submission_evaluation_status, if: -> { saved_change_to_completed_at? }
   after_destroy :update_submission_evaluation_status
 
+  def calculated_total_score(use_evaluator_scores: false)
+    total = if use_evaluator_scores
+              evaluation_scores.sum { |score| score.calculated_score(score.score) }
+            else
+              self[:total_score]
```

Comment:
nit: prefer attr_reader syntax
```suggestion
              self.total_score # (or `total_score` by itself also works)
```